deployment/templates/deploymentconfig.yaml

```diff
@@ -30,6 +30,27 @@ spec:
       - env:
         - name: LODESTAR_STATUS_GIT_TAG
           value: {{ .Values.imageTag }}
+        envFrom:
+        - configMapRef:
+            name: status-configuration
```

Comment:
What happens here if `{{ .Values.namespaces }}` is empty and the configmap isn't created?
My concern is that something will fail, if that's not the case then this is all good.

Reply:
If the configmap is not created, there is a default value assigned, which just means no component statuses will be reported through the api.  It won't fail or stop the application, but the api response will have an empty component check list.  Would it be better to put an if around this section?  i.e. {{- if .Values.namespaces }}, then set the configmap in the deploymentconfig?

Reply:
I'm good with this as is as long as it works as expected when the configmap isn't created (`.Values.namespaces=false`) which it sounds like it is.
Will mark as resolved.